ci/start_init_test.sh

```diff
@@ -342,6 +342,34 @@ get_replica_count() {
 	echo "$replicaCount"
 }
 
+verify_delete_replica() {
+    old_replica_count=$(get_replica_count $CONTROLLER_IP)
+    echo "$old_replica_count"
+    curl -X "POST" http://$CONTROLLER_IP:9501/v1/delete | jq
+    new_replica_count=$(get_replica_count $CONTROLLER_IP)
+    echo "$new_replica_count"
+    verify_replica_cnt "0" "Zero replica count test"
+}
+
+test_two_replica_delete() {
+	echo "----------------Test_two_replica_delete--------------"
+	orig_controller_id=$(start_controller "$CONTROLLER_IP" "store1" "2")
+	replica1_id=$(start_replica "$CONTROLLER_IP" "$REPLICA_IP1" "vol1")
+	replica2_id=$(start_replica "$CONTROLLER_IP" "$REPLICA_IP2" "vol2")
+	sleep 5
+	verify_replica_cnt "2" "Two replica count test1"
+	# This will delay sync between replicas
+	run_ios_to_test_stop_start
+	verify_delete_replica "Delete replicas test2"
+	sleep 5
+
+	docker stop $replica1_id
+	docker stop $replica2_id
+	docker stop $orig_controller_id
+	cleanup
+}
+
```

Comment:
Can we also test failures of deleting lets say 1 out of 3 replicas? -- and what happens if this is the case?

Reply:
yeah sure

Reply:
Added the negative test case where one of the replica goes down and sending a delete request i get a response like this.
```
{
  "actions": {},
  "links": {
    "self": "http://172.18.0.2:9501/v1/delete"
  },
  "replicas": [
    {
      "msg": "Replica deleted successfully",
      "replica": "tcp://172.18.0.4:9502"
    },
    {
      "error": {
        "Err": {},
        "Op": "Get",
        "URL": "http://172.18.0.3:9502/v1/replicas/1"
      },
      "msg": "Error deleting replica",
      "replica": "tcp://172.18.0.3:9502"
    }
  ],
  "type": "delete"
}
```